docs/docs/dev_docs/contracts/events.md

```diff
@@ -1,17 +1,104 @@
 ## Events
+Events in Aztec works similarly to Ethereum events in a sense that they are a way for contracts to communicate with the outside world.
+They are emitted by contracts and stored inside each instance ofAztecNode.
+Aztec events are currently represented as raw data and are not ABI encoded.
+ABI encoded events are a feature that will be added in the future.
```

Comment:
```suggestion
> Aztec events are currently represented as raw data and are not ABI encoded.
> ABI encoded events are a feature that will be added in the future.
```

I like to put information which talks about our future plans, protocol limitations, or justifies our decisions in 'quote indents' (or whatever they're called). Basically, any time you 'break the 4th wall'.